code/game/objects/items/storage/secure.dm

```diff
@@ -28,31 +28,29 @@
 
 /obj/item/storage/secure/examine(mob/user)
 	..()
-	to_chat(user, text("The service panel is [src.open ? "open" : "closed"]."))
+	to_chat(user, text("The service panel is currently <b>[open ? "unscrewed" : "screwed shut"]</b>."))
 
 /obj/item/storage/secure/attackby(obj/item/W, mob/user, params)
 	if(locked)
 		if (istype(W, /obj/item/screwdriver))
 			if (do_after(user, 20*W.toolspeed, target = src))
-				src.open =! src.open
-				user.show_message("<span class='notice'>You [open ? "open" : "close"] the service panel.</span>", 1)
+				open =! open
+				to_chat(user, "<span class='notice'>You [open ? "open" : "close"] the service panel.</span>")
 			return
-		if ((istype(W, /obj/item/device/multitool)) && (src.open == 1)&& (!src.l_hacking))
-			user.show_message("<span class='danger'>Now attempting to reset internal memory, please hold.</span>", 1)
-			src.l_hacking = 1
-			if (do_after(usr, 100*W.toolspeed, target = src))
-				if (prob(33))
-					src.l_setshort = 1
+		if (istype(W, /obj/item/wirecutters) || istype(W, /obj/item/card/emag))
+			to_chat(user, "<span class='danger'>[src] is protected from this sort of tampering, perhaps a <b>multitool</b> could pulse the wires controlling internal memory instead?</span>")
+		if ((istype(W, /obj/item/device/multitool)) && (!l_hacking))
+			if(src.open == 1)
+				to_chat(user, "<span class='danger'>Now attempting to reset internal memory, please hold.</span>")
+				src.l_hacking = 1
+				if (do_after(usr, 400*W.toolspeed, target = src))

```

Comment:
since the handheld briefcases inherit from this it would be wise to have `target = user` instead imo else you won't necessarily see the progress bar if you have it inhand.

Reply:
Yea I think I changed that in my prior PR, targeting the item doesn't even seem to give an inhand bar anymore.